pkg/autodetect/operatorconfig.go

```diff
@@ -24,6 +24,21 @@ func (p Platform) String() string {
 	return [...]string{"Kubernetes", "OpenShift"}[p]
 }
 
+// ESOperatorIntegration holds the if the ES Operator integration is enabled.
+type ESOperatorIntegration int
+
+const (
+	// ESOperatorIntegrationYes represents the ES Operator integration is enabled.
+	ESOperatorIntegrationYes ESOperatorIntegration = iota
```

Comment:
Does it make sense to start here with an `ESOperatorIntegrationUnknown` value?

Reply:
I don't think it is needed